apiml-security-common/src/main/resources/security-common-log-messages.yml

```diff
@@ -88,6 +88,41 @@ messages:
     # TLS,Certificate messages
     # 500-599
 
+    - key: org.zowe.apiml.security.common.filter.errorParsingCertificate
+      number: ZWEAT500
+      type: ERROR
+      text: "Failed to parse the client certificate forwarded from the central gateway. Error message %s. The client certificate was %s"
+      reason: "The string sent by the central gateway was not recognized as valid DER-encoded certificate in the Base64 printable form."
+      action: "Ensure that the forwarding of client certificate is enabled also in the central gateway. Check for any error messages in the central gateway."
```

Comment:
should we also specify in the action to check for valid DER-encoded certificates in the Base64 printable form, similarly to what it's stated in the `org.zowe.apiml.security.common.verify.errorParsingCertificates` message?

Reply:
fixed